docs/operations/ces_installation_de.md

```diff
@@ -107,12 +107,17 @@ Beispiel für einen Cluster aus einem Main-Node und drei Worker-Nodes:
          "name":"ces-worker-2",
          "node-ip":"192.168.2.102",
          "node-external-ip":"192.168.2.102",
-         "flannel-iface":"eth0"
+         "flannel-iface":"eth0",
+         "node-labels": ["foo=bar", "foo/bar.io=muh"],
+         "node-taints": ["key1=value1:NoExecute"]
       }
    ]
 }
 ```
 
+> Info: Die verwendeten Node-Labels und -Taints sind optional und können pro Node konfiguriert werden.
+> Weitere Hinweise zur Verwendung sind [hier](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) zu finden.
```

Comment:
Mayba add a link for node labels too? like this one? https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/